src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -1851,6 +1851,15 @@ class ongoingTasksStats extends viewModelBase {
                     case "Sql":
                     case "Olap": {
                         const elementWithData = context.rootStats as EtlPerformanceBaseWithCache;
+                        
+                        if (elementWithData.HasTransformErrors) {
+                            tooltipHtml += `<div class="tooltip-li text-danger">ETL task has Transform errors:<div class="value">Open notification center for details </div></div>`;
```

Comment:
```suggestion
                            tooltipHtml += `<div class="tooltip-li text-danger">ETL task has Transform errors:<div class="value">Open Notification center for details </div></div>`;
```

Reply:
For naming consistency:

![image](https://user-images.githubusercontent.com/376551/124071826-3fea1680-da40-11eb-8da1-f53c1278c875.png)

Reply:
done, unified all places to `Notification Center`